packages/kbn-build/cli.js

```diff
@@ -0,0 +1,2 @@
+require("./dist/cli").run(process.argv.slice(2));
```

Comment:
__nit:__ For some reason you've started to use double-quotes instead of single quotes everywhere in this PR :)

Reply:
Ah, yeah, I just forgot to set `--single-quote` for Prettier. Fixed it now.